function async(callback) {
  let result
  try {
    // .........
  } catch (error) {
    callback(error)
  }
  callback(null, result)
}
